state/dbstore.go

```diff
@@ -107,14 +107,26 @@ func (s *DBStore) Delete(key string) (err error) {
 	return s.db.Delete([]byte(key), nil)
 }
 
-// Iterate entries which has a specific prefix
-// The key and value may be modified in the iterFunc
-func (s *DBStore) Iterate(prefix string, iterFunc func([]byte, []byte)) (err error) {
+// iterFunction is a function called on every key/value pair obtained
```

Comment:
also in another PR, https://github.com/ethersphere/swarm/pull/1649#discussion_r313806607

Reply:
thanks for the heads-up @zelig! it seems like there won't be any conflicts merging these.

I've gone ahead and applied your named variables suggestion from that PR :+1: